Update Board/Officer members for 2020

JenHomann opened the pull request:
Update the board members, officers, and volunteers for 2020, per issue #72. I tried to follow what appears to be existing patterns:

- officers listed first for the board (in the same order as before: P, VP, S, T)
- board members and advisory board members listed in alphabetical order
- volunteers weren't alphabetical ... so I just put Eris alphabetical with the other volunteer with 3 tags 🤷‍♀
- I also added organizer tags for the rest of the board members who didn't have it?

Board & officers:
<img width="1677" alt="Screen Shot 2020-03-11 at 8 52 30 PM" src="https://user-images.githubusercontent.com/7945936/76479782-e9843f00-63da-11ea-8027-81aadda2dfe5.png">

Volunteers:
![image](https://user-images.githubusercontent.com/7945936/76479837-0c165800-63db-11ea-989a-0436d9bb5a85.png)

JenHomann commented on volunteer.html:
```diff
@@ -79,7 +79,7 @@
     </li>
   </nav>
-  
+
```
My IDE keeps editing the white space no matter how many times I revert these changes. I can open it in a new IDE if it bothers y'all too much. But for the short turnaround, this doesn't bother me.

Gmfholley commented on volunteer.html:
```diff
@@ -165,45 +163,50 @@ <h4>Treasurer</h4>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/morgan.jpg" alt="Morgan Smith headshot">
-            <h3>Morgan Smith</h3>
+            <img class="bio-pic" src="assets/images/volunteers/biancaz.jpg">
```
Could we add an alt tag here too?

Gmfholley commented on volunteer.html:
```diff
@@ -165,45 +163,50 @@ <h4>Treasurer</h4>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/Erisheadshot.jpg" alt="Eris Koleszar headshot">
-            <h3>Eris Koleszar</h3>
+            <img class="bio-pic" src="assets/images/volunteers/hauwa.jpg">
```
@JenHomann Could we add an alt tag here too?

Gmfholley commented on volunteer.html:
```diff
@@ -165,45 +163,50 @@ <h4>Treasurer</h4>
-            <img class="bio-pic" src="assets/images/volunteers/morgan.jpg" alt="Morgan Smith headshot">
-            <h3>Morgan Smith</h3>
+            <img class="bio-pic" src="assets/images/volunteers/biancaz.jpg">
+            <h3>Bianca Zongrone</h3>
```
I failed to mention this, but Bianca got married, and her married name is Bianca Zongrone Jefferson.  While you're here, could you make this change?

Sorry for not mentioning it earlier.

Gmfholley commented on volunteer.html:
```diff
@@ -222,31 +225,44 @@ <h2>Advisory Board</h2>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/biancaz.jpg">
-            <h3>Bianca Zongrone</h3>
+            <img class="bio-pic" src="assets/images/volunteers/jenh.jpg">
```
I see we weren't very good at adding alt tags before.  😓  Could you add one here, as well?

Thank you!

JenHomann commented:
> Thank you so much for this quick turnaround.
> 
> A few small changes! If you can't get to this in a timely way, let's merge for now and create a new issue.
> 
> Thank you so much for your work on this!

Whoops. Good call! Updated with alt tags. Went ahead and added them for folks that we didn't touch in this change and changed them all to the name + headshot format for consistency.

Gmfholley commented:
Thank you @JenHomann and @mel-d (who did some code review last night too).

This looks good.  Merging.
